lib/game.rb

```diff
@@ -0,0 +1,89 @@
+class Game
+  attr_accessor :player_1, :player_2, :board
+
+  WIN_COMBINATIONS = [[0, 1, 2],
+                      [3, 4, 5],
+                      [6, 7, 8],
+                      [0, 3, 6],
+                      [1, 4, 7],
+                      [2, 5, 8],
+                      [0, 4, 8],
+                      [2, 4, 6]]
+
+  def initialize(player_1 =  Players::Human.new("X"), player_2 =  Players::Human.new("O"), board =  Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    if board.turn_count.even?
+      self.player_1
+    elsif board.turn_count.odd?
+      self.player_2
+    end
+  end
+
+  def over?
+    board.full?
+  end
+
+  def won?
+    WIN_COMBINATIONS.detect do |win_combo|
+      if win_combo.all? {|cell| self.board.cells[cell] == "X"} || win_combo.all? {|cell| self.board.cells[cell] == "O"}
+        win_combo
+      else
+        false
+      end
+    end
+  end
+
+  def draw?
+    self.over? && !self.won?
+  end
+
+  def winner
+    if won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.